src/main/java/htsjdk/tribble/gff/Gff3Codec.java

```diff
@@ -198,6 +180,31 @@ public Gff3Feature decode(final LineIterator lineIterator) throws IOException {
         return attributes;
     }
 
+    static private Gff3BaseData parseLine(final String line) {
+        final List<String> splitLine = ParsingUtils.split(line, FIELD_DELIMITER);
+
+        if (splitLine.size() != NUM_FIELDS) {
+            throw new TribbleException("Found an invalid number of columns in the given Gff3 file- Given: " + splitLine.size() + " Expected: " + NUM_FIELDS + " : " + line);
```

Comment:
I would pass the line number in here still.  It's useful enough for debugging that I think it's worth the cruft.